spec/student_spec.rb

```diff
@@ -0,0 +1,27 @@
+require_relative '../student'
```

Comment:
Please add the `bundle install` command in README so a user can install project dependencies.
Please add `rspec` command in README so a user can run the test.